pandas/tests/io/formats/test_to_csv.py

```diff
@@ -253,3 +253,12 @@ def test_to_csv_string_array_utf8(self):
             df.to_csv(path, encoding='utf-8')
             with open(path, 'r') as f:
                 assert f.read() == expected_utf8
+
+    def test_to_csv_sparse_dataframe(self):
+        sdf = pd.SparseDataFrame({'a': [1, 2]})
```

Comment:
add a comment here for the issue number. I would rather have this test in pandas/tests/sparse/frame/test_to_csv.py 

can you add several variations, e.g. using fill value and with some nulls.